sw/device/tests/sim_dv/uart_tx_rx_test.c

```diff
@@ -92,6 +117,101 @@ static volatile bool uart_irq_tx_empty_fired;
 static volatile bool exp_uart_irq_rx_overflow;
 static volatile bool uart_irq_rx_overflow_fired;
 
+// TODO, remove this once pinout configuration is provided
+static void init_uart_pins(uint32_t rx_pad, uint32_t rx_uart, uint32_t tx_pad,
```

Comment:
it's good to keep the `uart`, so that we know if it's a pinmux offset or uart offset. Changed as follows.
rx_pad -> rx_pin_out_idx
rx_uart -> rx_uart_idx